input/netmap/netmap.c

```diff
@@ -891,6 +1552,18 @@ static int open_netmap( netmap_ctx_t *netmap_ctx )
         upipe_release(vbi);
     }
 
+    /* vbi filter */
+    struct upipe_mgr *upipe_filter_vbi_mgr = upipe_vbi_mgr_alloc();
+    struct upipe *probe_vbi = upipe_vbi_alloc_output(vbi,
+            upipe_filter_vbi_mgr,
+            uprobe_pfx_alloc(uprobe_use(uprobe_dejitter), loglevel, "vbi filter"),
+            uprobe_pfx_alloc(uprobe_obe_alloc(uprobe_use(uprobe_dejitter), catch_ttx, netmap_ctx),
+               UPROBE_LOG_DEBUG, "op47"),
```

Comment:
op47 is not the same as vbi

Reply:
Seems to work though

Reply:
Do you want to change the debug "op47" into "ttx" ?

Reply:
vbi_ttx